mlflow/tracking/llm_utils.py

```diff
@@ -0,0 +1,90 @@
+import logging
+from typing import Dict, List, Union
+
+_logger = logging.getLogger(__name__)
+
+
+def log_predictions(
+    inputs: List[Union[str, Dict[str, str]]],
+    outputs: List[str],
+    prompts: List[Union[str, Dict[str, str]]],
```

Comment:
Do we need an optional `artifact_path` that allows a user to log `llm_predictions.csv` in a subdirectory?

Reply:
Yes I thought about it. I even thought if users can change the artifact_file as well with the path. 
But the downside is that when rendering these files for the compare view, the FE code needs to read these files and build that compare view. 
If the path is dynamic, we would need to write a path or something to a tag where the FE can read the tag, get the llm_path and use that to create the compare view. Thoughts?

Reply:
Discussed offline in the sync meeting where we agreed to not have this and hardcode this instead. We can add this later if needed